<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ComputerDialog</class>
 <widget class="QDialog" name="ComputerDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>414</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Computer Properties</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_name">
         <property name="text">
          <string>Name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_parent_name">
         <property name="text">
          <string>Parent Group:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_address">
         <property name="text">
          <string>Address:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_port">
         <property name="text">
          <string>Port:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_username">
         <property name="text">
          <string>User Name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_password">
         <property name="text">
          <string>Password:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_session_config">
         <property name="text">
          <string>Session Configuration:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLineEdit" name="edit_name">
         <property name="maxLength">
          <number>64</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edit_parent_name">
         <property name="maxLength">
          <number>64</number>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edit_address">
         <property name="maxLength">
          <number>64</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinbox_port">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>65534</number>
         </property>
         <property name="value">
          <number>8050</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edit_username">
         <property name="maxLength">
          <number>64</number>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLineEdit" name="edit_password">
           <property name="maxLength">
            <number>64</number>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="button_show_password">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="console.qrc">
             <normaloff>:/icon/eye.png</normaloff>:/icon/eye.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QComboBox" name="combo_session_config"/>
         </item>
         <item>
          <widget class="QPushButton" name="button_session_config">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="console.qrc">
             <normaloff>:/icon/settings.png</normaloff>:/icon/settings.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_comment">
       <property name="text">
        <string>Comment:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="edit_comment"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="button_box">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="console.qrc"/>
 </resources>
 <connections/>
</ui>
